todo_app/app.py

```diff
@@ -1,67 +1,65 @@
 import os
+from todo_app.item import Item
 from flask import Flask, redirect, url_for,  render_template
 from todo_app.data import session_items
 from todo_app.flask_config import Config
 from flask import request
 import requests
+from todo_app.view_model import ViewModel
 
-app = Flask(__name__)
-app.config.from_object(Config)   
+def create_app():
+   app = Flask(__name__)
+   app.config.from_object(Config())   
 
-KEY=os.getenv("KEY")
-TOKEN=os.getenv("TOKEN")
+   KEY=os.getenv("KEY")
+   TOKEN=os.getenv("TOKEN")
 
-BOARD_ID=os.getenv("BOARD_ID")
-ID_LIST_TODO=os.getenv("ID_LIST_TODO")
-ID_LIST_DOING=os.getenv("ID_LIST_DOING")
-ID_LIST_DONE=os.getenv("ID_LIST_DONE")
+   BOARD_ID=os.getenv("BOARD_ID")
+   ID_LIST_TODO=os.getenv("ID_LIST_TODO")
+   ID_LIST_DOING=os.getenv("ID_LIST_DOING")
+   ID_LIST_DONE=os.getenv("ID_LIST_DONE")
 
+   @app.route('/index')
+   def index():
```

Comment:
Traditionally "index" means the route `/`. If you'd like to use `/index` that's fine, but it would be good to set up a redirect from `/` to `/index`

Reply:
Done